slices/tzdata.yaml

```diff
@@ -0,0 +1,300 @@
+      /usr/share/zoneinfo/Cuba:
+      /usr/share/zoneinfo/Jamaica:
+      /usr/share/zoneinfo/Navajo:
+      /usr/share/zoneinfo/posix/Cuba:
+      /usr/share/zoneinfo/right/Cuba:
+      /usr/share/zoneinfo/posix/Jamaica:
+      /usr/share/zoneinfo/right/Jamaica:
+      /usr/share/zoneinfo/posix/Navajo:
+      /usr/share/zoneinfo/right/Navajo:
+      /usr/share/zoneinfo/posixrules:
+
+  antarctica:
+    essential:
+      - tzdata_eurasia
+      - tzdata_pacific
+    contents:
+      /usr/share/zoneinfo/Antarctica/*:
+      /usr/share/zoneinfo/posix/Antarctica/*:
+      /usr/share/zoneinfo/right/Antarctica/*:
+
+  arctic:
+    essential:
+      - tzdata_eurasia
+    contents:
+      /usr/share/zoneinfo/Arctic/*:
+      /usr/share/zoneinfo/posix/Arctic/*:
+      /usr/share/zoneinfo/right/Arctic/*:
+
+  atlantic:
+    essential:
+      - tzdata_africa
+      - tzdata_eurasia
+    contents:
+      /usr/share/zoneinfo/Atlantic/*:
+      /usr/share/zoneinfo/posix/Atlantic/*:
+      /usr/share/zoneinfo/right/Atlantic/*:
+
+  australia:
+    contents:
+      /usr/share/zoneinfo/Australia/*:
+      /usr/share/zoneinfo/posix/Australia/*:
+      /usr/share/zoneinfo/right/Australia/*:
+
+  brazil:
+    essential:
+      - tzdata_america
+    contents:
+      /usr/share/zoneinfo/Brazil/*:
+      /usr/share/zoneinfo/posix/Brazil/*:
+      /usr/share/zoneinfo/right/Brazil/*:
+
+  canada:
+    essential:
+      - tzdata_america
+    contents:
+      /usr/share/zoneinfo/Canada/*:
+      /usr/share/zoneinfo/posix/Canada/*:
+      /usr/share/zoneinfo/right/Canada/*:
+
+  chile:
+    essential:
+      - tzdata_america
+      - tzdata_pacific
+    contents:
+      /usr/share/zoneinfo/Chile/*:
+      /usr/share/zoneinfo/posix/Chile/*:
+      /usr/share/zoneinfo/right/Chile/*:
+
+  etc:
+    contents:
+      /usr/share/zoneinfo/Etc/*:
+      /usr/share/zoneinfo/posix/Etc/*:
+      /usr/share/zoneinfo/right/Etc/*:
+      /usr/share/zoneinfo/GMT:
+      /usr/share/zoneinfo/GMT+0:
+      /usr/share/zoneinfo/GMT-0:
+      /usr/share/zoneinfo/GMT0:
+      /usr/share/zoneinfo/Greenwich:
+      /usr/share/zoneinfo/posix/GMT:
+      /usr/share/zoneinfo/posix/GMT+0:
+      /usr/share/zoneinfo/posix/GMT-0:
+      /usr/share/zoneinfo/posix/GMT0:
+      /usr/share/zoneinfo/posix/Greenwich:
+      /usr/share/zoneinfo/right/GMT:
+      /usr/share/zoneinfo/right/GMT+0:
+      /usr/share/zoneinfo/right/GMT-0:
+      /usr/share/zoneinfo/right/GMT0:
+      /usr/share/zoneinfo/right/Greenwich:
+      /usr/share/zoneinfo/posix/UCT:
+      /usr/share/zoneinfo/posix/Universal:
+      /usr/share/zoneinfo/posix/UTC:
+      /usr/share/zoneinfo/right/UCT:
+      /usr/share/zoneinfo/right/Universal:
+      /usr/share/zoneinfo/right/UTC:
+      /usr/share/zoneinfo/posix/Zulu:
+      /usr/share/zoneinfo/right/Zulu:
+      /usr/share/zoneinfo/UCT:
+      /usr/share/zoneinfo/Universal:
+      /usr/share/zoneinfo/UTC:
+      /usr/share/zoneinfo/Zulu:
```

Comment:
I would merge this with `base`. This contains `UTC` among other things. And I noticed `CET` is in `base`. It feels logical to group them together.

Reply:
So, `Etc` is meant to provide "timezones" that don't fit with the standard timezones. UTC isn't actually a timezone, but a standard. Like Zulu and others, most of these can be used for time information, but derive from different domains (like the military). Same for others.

This [table](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List) is actually useful as it notes that Zulu, UTC, and others within this slice are actually links and not Canonical timezones. 

I'll add comments to these slices to provide more info.